tests/acceptance/features/apiSpaces/moveSpaces.feature

```diff
@@ -0,0 +1,234 @@
+@api @skipOnOcV10
+Feature: move (rename) file
+  As a user
+  I want to be able to move and rename files
+  So that I can manage my file system
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+  Scenario Outline: Moving a file within same space project with role manager and editor
```

Comment:
```suggestion
  Scenario Outline: Moving a file within same project space with role manager and editor
```

"space project" is in a lot of feature files already - I will make a separate PR to discuss fixing that word order.